x-pack/plugins/license_management/public/sections/license_dashboard/request_trial_extension/request_trial_extension.js

```diff
@@ -22,7 +22,7 @@ export const RequestTrialExtension = ({ shouldShowRequestTrialExtension }) => {
         values={{
           platinumLicenseFeaturesLinkText: (
             <EuiLink
-              href="https://www.elastic.co/subscriptions/xpack"
+              href="https://www.elastic.co/subscriptions"
```

Comment:
We should probably make this link a constant since it appears in four places, so if it changes again, we only have to change the constant.

Reply:
good point. i'll update this.

Reply:
@bmcconaghy I went ahead and extracted external links into a constants file, if you could take another look when you get a chance. Thanks!